### ISSUE TITLE
[A11Y] Android: Unable to move focus on Social Media SHARE buttons present under Read More section on Article template

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#382]
**ENVIRONMENT:**

OS (number): N (7.1.2)
Build number: N2G47H
Form factor (Smartphone / tablet):  Nexus 6P
Browser: Chrome Dev 59.0.3056.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly navigate to the “Article” template and double tap on View Template
3. Linearly navigate to READ MORE section and try to move focus on social media buttons using linear navigation or exploratory touch.

**ACTUAL RESULTS:**

User is not able to move focus on social media share buttons/links.

**EXPECTED RESULTS:**

User should be able to move focus on social media share buttons/links and should be able to click on it using double tap action. The social media share buttons should have appropriate labels and controls assigned to it.

**ATTACHMENT(S):**

<img width="1440" alt="ampstart_socialbox" src="https://cloud.githubusercontent.com/assets/26986019/24823336/17a5523a-1bb2-11e7-9e2f-130309d42b20.png">

**Comment by gkhadke** [comment]
Same issue exists on the Components page as well.

**Comment by torch2424** [comment]
I was able to navigate using Talkback to the buttons on both fields, however, I opened a PR as they were not properly labeled.
